sql-db/multiple-pus/src/main/resources/import-fruits.sql

```diff
@@ -5,4 +5,4 @@ INSERT INTO fruit (id, name) VALUES (4, 'Plum');
 INSERT INTO fruit (id, name) VALUES (5, 'Cherry');
```

Comment:
Could be possible to replace these hardcoded IDs with something like `nextval('fruit_SEQ'` ?

Reply:
yes